packages/react-select/src/NonceProvider.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import React, { Component, type Node } from 'react';
-import { CacheProvider } from '@emotion/core';
+import { CacheProvider } from '@emotion/react';
 import createCache from '@emotion/cache';
```

Comment:
As for the new required `key` option in `createCache`, I would propose we just add it to the `props` of `NonceProvider` and make the user provide it. I'm not thinking of any good ways to deal with it on our end. We're going to be releasing this PR in a major release anyway.

Reply:
I suppose we could do something similar to an auto-incrementing `instanceId` that we have for `Select`, but I think it'd be preferable to have the user handle the `key`.